patientMatcher/match/handler.py

```diff
@@ -252,7 +253,7 @@ def async_match(database, response_data):
     resp = response_data.get('response')
     results_obj = {
         'node' : node,
-        'patients' : resp.get('results')
+        'patients' : resp.get('results', [])
```

Comment:
It was quite hard for me to debug because I didn't have your server in place to test the function. Nice that you fixed it!

Reply:
Yeah, same for me. Finally being able hook them up together found a lot of things to fix on my end. :laughing:

Reply:
I'm sure we will discover other bugs in the future. Hopefully after Tuesday!